provider/types.go

```diff
@@ -13,3 +13,7 @@ type Status struct {
 	Manifest              *manifest.Status  `json:"manifest"`
 	ClusterPublicHostname string            `json:"cluster_public_hostname,omitempty"`
 }
+
+type Validate struct {
```

Comment:
Maybe `ValidateResult` is a better name?

Reply:
`ValidateGroupSpecResult` ?